lib/android/audio.nit

```diff
@@ -14,54 +14,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Android audio services
+# Android audio services, wraps a part of android audio API
 #
-# You can get a sound by loading it with a `SoundPool` or a `MediaPlayer`
-# the recommended way to load a sound is by it's resource ID or it's name
-# other ways are for advanced use
+# Use `load_sound`, `load_music`, `load_sound_from_res`
+# or `load_music_from_res` to get a `Sound`.
+# Also get a sound by loading it with a `SoundPool` or a `MediaPlayer`
```

Comment:
I dont't understand the `Also`.

Is this something I have to do or something done by one one these three methods?